paws_collector.js

```diff
@@ -303,6 +396,9 @@ class PawsCollector extends AlAwsCollector {
             },
             function(privCollectorState, nextInvocationTimeout, asyncCallback) {
                 return collector._storeCollectionState(pawsState, privCollectorState, nextInvocationTimeout, asyncCallback);
+            },
+            function(_results, asyncCallback) {
+                return collector.updateStateDBEntry(stateSqsMsg, STATE_RECORD_COMPLETE, asyncCallback);
```

Comment:
I think we can move `updateStateDBEntry()` into the function above and use `async.parallel` to run them independently. Ideas?

Reply:
We need to make sure the SqS push succeeds before we update the DDB state.

If we run them in paralelle, the DDB call might succeed where the SQS push might fail. If that happens, the retry of the state will be marked as a duplicate, and stop the collector.